# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

SetptActionGoal action_goal
SetptActionResult action_result
SetptActionFeedback action_feedback
